bliss/predict.py

```diff
@@ -33,8 +36,46 @@ def predict_sdss(cfg):
     return est_cat, images[0], background[0], sdss_plocs, pred
 
 
-def decal_plocs_from_sdss(cfg):
-    """Use wcs to match sdss image with corresponding DeCals catalog for futher plotting."""
+def decals_plocs_from_decals(decals: DarkEnergyCameraLegacySurvey, cfg):
+    # get ra_lim, dec_lim corresponding to DECaLS image(s)
+    width_in_deg = DarkEnergyCameraLegacySurvey.pix_to_deg(cfg.predict.dataset.width)
+    height_in_deg = DarkEnergyCameraLegacySurvey.pix_to_deg(cfg.predict.dataset.height)
+    ra_lim = (
+        cfg.predict.dataset.ra - width_in_deg / 2,
+        cfg.predict.dataset.ra + width_in_deg / 2,
+    )
+    dec_lim = (
+        cfg.predict.dataset.dec - height_in_deg / 2,
+        cfg.predict.dataset.dec + height_in_deg / 2,
+    )
+    tractor_filename = cfg.paths.decals + f"/tractor-{decals.get_brick_name()}.fits"
+    return DecalsFullCatalog.from_file(tractor_filename, ra_lim, dec_lim).plocs[0]
+
+
+def predict_decals(cfg):
```

Comment:
Can we merge the `predict_sdss(...)` and `predict_decals(...)` functions? Ideally the survey-specific routines are all abstracted away, stored in `sdss.py` and `decals.py`.

Reply:
Tracked by #853.